video_xblock/video_xblock.py

```diff
@@ -209,3 +320,48 @@ def get_player(self):
         """
         player = BaseVideoPlayer.load_class(self.player_name)
         return player()
+
+    def _make_field_info(self, field_name, field):
+        """
+        Overrides and extends data of built-in method
+        """
+        if field_name in ('start_time', 'end_time'):
+            # RelativeTime field doesn't supported by default.
+            return {
```

Comment:
what it returns by default (without this `if field_name in ('start_time', 'end_time')`)?